backend/main.py

```diff
@@ -1,147 +1,17 @@
 from fastapi import FastAPI
 
+import json
+
+with open('../database/bobalist.json') as f:

```

Comment:
put this code block under `async def get_boba_list():`

Reply:
makes sense . noted.